src/queries/types/GetAccessibilityRequest.ts

```diff
@@ -0,0 +1,23 @@
+/* tslint:disable */
+/* eslint-disable */
+// @generated
```

Comment:
Can we put generated code in a folder called `generated`? That's helpful on the backend at least for keeping track in the file structure of what I'm allowed to edit.

Reply:
I'm on board for this change, but let's do it outside this story since the pattern was already in place.